Tiltfile

```diff
@@ -396,6 +401,77 @@ def prepare_all():
     )
     local(cmd, env = settings.get("kustomize_substitutions", {}))
 
+# create flavor resources from cluster-template files in the templates directory
+def flavors():
+    substitutions = settings.get("kustomize_substitutions", {})
+
+    template_list = [item for item in listdir("./test/infrastructure/docker/templates")]
+    template_list = [template for template in template_list if os.path.basename(template).endswith("yaml")]
+
+    for template in template_list:
+        deploy_worker_templates(template, substitutions)
+
+    local_resource(
+        name = "delete-all-workload-clusters",
+        cmd = kubectl_cmd + " delete clusters --all --wait=false",
+        auto_init = False,
+        trigger_mode = TRIGGER_MODE_MANUAL,
+        labels = ["flavors"],
+    )
```

Comment:
I think that having this as a resource along with the templates is kind of awkward and I see what you mean since the command isn't very complicated and doesn't use envsubst. I added it in CAPZ originally with @CecileRobertMichon since we figured it would be easy to tear down everything if you've spun up dozens of clusters.